FrontEnd/src/Components/ModalForm.jsx

```diff
@@ -90,162 +95,22 @@ function ModalForm({ onClose }) {
-                        placeholder="Writing Assistant..."
-                      />
-                      <Text fontSize="14px" color="#B3B3B3" textAlign="right">
-                        0/40
-                      </Text>
-                    </FormControl>
-                  </Flex>
-                  <FormControl id="promptDescription" mt="24px">
-                    <Box mb="12px">
-                      <Text fontSize="20px" fontWeight="semibold">
-                        Prompt Description
-                      </Text>
-                      <Text fontSize="14px" fontStyle="italic" color="#B3B3B3">
-                        Describe what your prompt does to a potential user
-                      </Text>
-                    </Box>
-                    <Textarea
-                      name="promptDescription"
-                      value={formValues.promptDescription}
-                      onChange={handleInputChange}
-                      color="#333333"
-                      bg="white"
-                      borderRadius="4px"
-                    />
-                    <Text fontSize="14px" color="#B3B3B3" textAlign="right">
-                      0/500
-                    </Text>
-                  </FormControl>
-                  <Flex
-                    justifyContent="space-between"
-                    alignItems="center"
-                    mt="24px"
-                  >
-                    <Box>
-                      <Text fontSize="14px" fontStyle="italic" color="#B3B3B3">
-                        *All the Fields are Mandatory to fill
-                      </Text>
-                    </Box>
-                    <Box justifyContent="space-between">
-                      <Button
-                        bg="#E5E3FF"
-                        color="#6C63FF"
-                        borderRadius="3px"
-                        onClick={handleCloseModal}
-                        mr="30px"
-                        _hover={{}}
-                        disabled={
-                          !formValues.promptType ||
-                          !formValues.promptName ||
-                          !formValues.promptDescription
-                        }
-                      >
-                        Cancel
-                      </Button>
-                      <Button
-                        bg="#6C63FF"
-                        color="white"
-                        borderRadius="3px"
-                        onClick={() => setPage(2)}
-                        _hover={{}}
-                        disabled={
-                          !formValues.promptType ||
-                          !formValues.promptName ||
-                          !formValues.promptDescription
-                        }
-                      >
-                        Next
-                      </Button>
-                    </Box>
-                  </Flex>
-                </Box>
+                <Form1
```

Comment:
Please use descriptive names for custom components

Reply:
Sure